ruby/bps-stan/spec/bps/stan_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'bps/stan'
+require 'spec_helper'
+
+NATS_CLUSTER = 'test-cluster'.freeze
+NATS_CLIENT  = 'bps-test'.freeze
+
+def nats_servers
+  ENV.fetch('NATS_SERVERS', '127.0.0.1:4222').split(',').freeze
+end
+
+def nats_servers_with_scheme
+  nats_servers.map {|s| "nats://#{s}" }
+end
+
+run_spec = \
```

Comment:
separate PR probably but this doesn't work well, I changed this to rspec tags on bfs.rb side, we should do the same here